src/Subtitles/SubtitleComponent.js

```diff
@@ -0,0 +1,203 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Lightning from '../Lightning'
+
+export default class SubtitleComponent extends Lightning.Component {
+  static _template() {
+    return {
+      visible: false,
+      rect: true,
+      color: 0x90000000,
+      shader: { type: Lightning.shaders.RoundedRectangle, radius: 5 },
+      Text: {
+        y: 5,
+        x: 20,
+        text: {
+          fontColor: 0xffffffff,
```

Comment:
This doesn't seem correct. Isn't the property `textColor`? [Related docs](https://github.com/rdkcentral/Lightning/blob/master/docs/RenderEngine/Textures/Text.md)

Reply:
Thanks, this also has been fixed.